Yoast/javascript
# Upgrade content analysis app to latest deps

## Summary

This PR can be summarized in the following changelog entry:

* Upgrades all dependencies in the content analysis test app.

## Relevant technical choices:

* To make sure everything works I have ejected a new `create-react-app` application and copied over the files and made sure our custom changes were still in place.

## Test instructions

This PR can be tested by following these steps:

* Run the content analysis app using `cd apps/content-analysis && yarn start`.
* Use all features to make sure they still work.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
